terraform/e2e/main.tf

```diff
@@ -15,6 +15,14 @@
 locals {
   mapping_service_name = "mapping"
   policy_service_name  = "policy"
+
+  alert_subscription_map = {
+    mapping_bigquery         = module.common_infra.bigquery_subscriptions[local.mapping_service_name].event_subscription_id,
+    mapping_bigquery_failure = module.common_infra.bigquery_subscriptions[local.mapping_service_name].failure_subscription_id,
+    policy_bigquery          = module.common_infra.bigquery_subscriptions[local.policy_service_name].failure_subscription_id,
```

Comment:
Should we have both policy_bigquery and policy_bigquery_failure?

Reply:
Added

Reply:
Did you add it?

Reply:
sorry it's somehow override by other changes, added now.